x/gov/simulation/operations.go

```diff
@@ -366,13 +398,24 @@ func SimulateMsgDeposit(
 }
 
 // SimulateMsgVote generates a MsgVote with random values.
+// Deprecated: this is an internal method and will be removed
```

Comment:
will these still be deprecated?

Reply:
Yes, I would be surprised if anybody is using these methods. They are public only for testing purpose to avoid cyclic dependencies.

One goal of simsx is a better dev UX by returning the relevant data instead of talking to the baseapp directly. https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/simulation/msg_factory.go#L43 for example
I would still recommend to go that path and drop the old methods that have this tight coupling and complexity.